sites/brussel/macros.tex

```diff
@@ -10,6 +10,7 @@
 \macro{\hpcname}{Hydra}
 \macro{\loginnode}{hydra.vub.ac.be}
 \macro{\loginhost}{nic50.hydra.vub.ac.be}
+\macro{\altloginhost}{nic51.hydra.vub.ac.be}
```

Comment:
use `.brussel.vsc` instead of `.vub.ac.be`

Reply:
Can you open a follow-up PR for that? I just copied what was there already for `\loginhost`...